static/app/utils/profiling/spanTree.tsx

```diff
@@ -50,15 +79,23 @@ class SpanTreeNode {
 }
 
 class SpanTree {
-  root: SpanTreeNode = SpanTreeNode.Root();
+  root: SpanTreeNode;
   orphanedSpans: RawSpanType[] = [];
 
-  constructor(spans: RawSpanType[]) {
+  constructor(transaction: EventTransaction, spans: RawSpanType[]) {
+    this.root = SpanTreeNode.Root({
+      description: transaction.title,
+      start_timestamp: transaction.startTimestamp,
+      timestamp: transaction.endTimestamp,
+      exclusive_time: transaction.endTimestamp - transaction.startTimestamp,
```

Comment:
This is just calculating the duration of the transaction and not the exclusive time. If you want the exclusive time, you can find it in the trace context.